tests/e2e/TestConstants.ts

```diff
@@ -149,6 +149,11 @@ export const TestConstants = {
      */
     TS_SELENIUM_REPORT_FOLDER: process.env.TS_SELENIUM_REPORT_FOLDER || './report',
 
+    /**
+     * Path to folder with load tests execution report.
+     */
+    TS_SELENIUM_LOAD_TEST_REPORT_FOLDER: process.env.TS_SELENIUM_LOAD_TEST_REPORT_FOLDER || './load-test-folder',
```

Comment:
Duplication of ```    TS_SELENIUM_REPORT_FOLDER: process.env.TS_SELENIUM_REPORT_FOLDER || './report',```

Reply:
I would not say it is duplication. It is just separation load test logs from the rest of the report. It seems more clear to me. Regarding this logic [1] when the test passes, the report folder is deleted, which is not something we want to do with the load test report. Currently, this folder contains only one file, but in the future, more files can be added there. 
@musienko-maxim you were author of this logic, WDYT?

[1] https://github.com/eclipse/che/blob/master/tests/e2e/driver/CheReporter.ts#L95

Reply:
On my opinion, we should try to use existing variables everywhere where it possible and don't increase their count. Just set the "DELETE_SCREENCAST_IF_TEST_PASS" to "false" and set the necessary report folder by "TS_SELENIUM_REPORT_FOLDER" variable.

Reply:
But if we do that, we will have a huge amount of screenshots. Just to have an idea - currently test is really small and its screenshots and reports can be around 30MB. When load tests would run with 50 users and e.g. two repetitions, it would be 3000MB. All those would need to be stored on a cluster before they'll be synced to your local machine. And the value of them is minimal.

Reply:
Ok, make sense.